assets/base.css

```diff
@@ -1534,21 +1567,23 @@ details[open] > .share-button__fallback {
   color: rgba(var(--color-foreground), 0.75);
 }
 
-.field__input:hover,
+.field:hover,
 .select__select:hover,
-.customer .field input:hover,
+.customer .field:hover,
 .customer select:hover,
 .localization-form__select:hover {
-  box-shadow: 0 0 0 calc(var(--inputs-border-width) + 1px) rgba(var(--color-foreground), var(--inputs-border-opacity));
+  box-shadow: 0 0 0 0.3rem rgba(var(--color-foreground), var(--inputs-border-opacity));
+  border-radius: var(--inputs-radius);
 }
```

Comment:
I'm finding the hover to look a bit off compared to the border of the element: [video](https://screenshot.click/22-01-xqr3i-wm4fs.mp4)

Also there is a transition I can see to apply the border radius to it: [video](https://screenshot.click/22-01-axajl-h8azh.mp4)

Reply:
I will be pairing with Melissa on the hover/focus states to see what makes the most sense 👍